src/containers/ContactManagement/UploadContactsDialog/UploadContactsDialog.tsx

```
@@ -26,12 +27,30 @@ export const UploadContactsDialog = ({ setDialog, setShowStatus }: UploadContact
   const [fileName, setFileName] = useState<string>('');
   const [csvContent, setCsvContent] = useState<String | null | ArrayBuffer>('');
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)